From b748b41985a77c6709596c98222f149514a61203 Mon Sep 17 00:00:00 2001
From: "duncan.macleod" <duncan.macleod@ligo.org>
Date: Mon, 17 Oct 2022 10:23:43 +0100
Subject: [PATCH 1/2] setup.py: update requirements

we don't need pytest-runner to build, or lalsuite on conda
---
 setup.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/setup.py b/setup.py
index 783978e..eade3e0 100755
--- a/setup.py
+++ b/setup.py
@@ -35,12 +35,10 @@ setup(
     ],
     packages=find_packages(),
     include_package_data=True,
-    setup_requires=['pytest-runner'],
     tests_require=['pytest'],
     namespace_packages=['ligo'],
     install_requires=[
         'astropy',
-        'lalsuite',
         'numpy',
         'python-ligo-lw',
         'scipy',
-- 
2.20.1

